### ISSUE TITLE
"Sum" reducer?

### ISSUE THREAD

**Issue by AngledLuffa** [KevinMusgrave/pytorch-metric-learning#553]
Is there a mechanism to do "sum" reduction over errors?

**Comment by KevinMusgrave** [comment]
It isn't built-in yet. For now you can use this:

```python
import torch
from pytorch_metric_learning.reducers import BaseReducer

class SumReducer(BaseReducer):
    def element_reduction(self, losses, *_):
        return torch.sum(losses)

    def pos_pair_reduction(self, losses, *args):
        return self.element_reduction(losses, *args)

    def neg_pair_reduction(self, losses, *args):
        return self.element_reduction(losses, *args)

    def triplet_reduction(self, losses, *args):
        return self.element_reduction(losses, *args)


reducer = SumReducer()
```

**Comment by AngledLuffa** [comment]
Thanks for the suggestion!

**Comment by KevinMusgrave** [comment]
Added in v1.7.1

**Comment by AngledLuffa** [comment]
Awesome, thanks!
